packages/web-components/src/global/color-mode.css

```diff
@@ -178,6 +178,49 @@ ic-theme,
   );
   --ic-status-tag-outlined-neutral: var(--ic-status-unknown-mid);
 
+  /* Stepper */
+  --ic-step-title: var(--ic-color-text-primary);
+  --ic-step-title-current: var(--ic-status-info-default);
+  --ic-step-title-success: var(--ic-status-success-default);
+  --ic-step-title-active: var(--ic-color-text-tertiary);
+  --ic-step-title-disabled: var(--ic-color-text-disabled-default);
+  --ic-step-title-status-text-current: var(--ic-color-text-primary);
+  --ic-step-title-status-text-inactive: var(--ic-color-text-tertiary);
+  --ic-step-title-status-text-disabled: var(--ic-color-text-disabled-default);
+  --ic-stepper-connector-success: var(--ic-status-success-default);
+  --ic-stepper-connector-current-status: var(--ic-status-info-default);
+  --ic-stepper-connector-remaining-status: var(--ic-architectural-200);
+  --ic-stepper-connector-disabled: var(--ic-architectural-200);
+  --ic-step-number-background-active: var(--ic-status-info-default);
+  --ic-step-number-background-success: var(--ic-status-success-default);
+  --ic-step-number-text: var(--ic-color-text-inverse);
+  --ic-step-number-text-inactive: var(--ic-color-text-primary);
+  --ic-step-number-text-disabled: var(--ic-color-text-disabled-default);
+  --ic-step-number-icon-success: var(--ic-color-icon-inverted);
+  --ic-step-number-inactive: var(--ic-color-background-primary);
+  --ic-step-number-disabled: var(--ic-architectural-200);
+  --ic-step-number-inactive-outline: var(--ic-architectural-200);
```

Comment:
Some of these values are pointing to primitive tokens i.e. architectural colors when they should be pointing to semantic tokens

Reply:
`--ic-step-number-inactive-outline` for dark mode doesn't seem to match up with the Figma designs

Reply:
They're pointing to architectural tokens in places that it looked like incorrect semantic tokens had been used e.g. hover tokens used for non-hover states